spec/features/workflow/schedule_requirements_spec.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+RSpec.feature "Scheduling requirements" do
+  scenario do
```

Comment:
I don't think we need to cover all the cases here - the pattern in the other requirements features is just to test the UI works when just one thing is wrong, and leave the rest to the unit spec for the checker.

Reply:
Now only testing one scenario